

float32 confidence

int32 detection

